prov/verbs/src/fi_verbs.c

```diff
@@ -77,6 +77,13 @@ struct fi_ibv_gl_data fi_ibv_gl_data = {
 	},
 };
 
+struct fi_ibv_verbs_prov_preset verbs_prov_presets[] = {
+	{
+		.max_inline_data = 48,
```

Comment:
Can you add a comment how we got the value of 48? Any reference to relevant documentation (if publicly available)  would be helpful.

Reply:
I've got this information form [GH repo](https://github.com/linux-rdma/rdma-core/blob/fb813d6316b620f08f7ff5d35d04d25e1200e24e/providers/i40iw/i40iw_user.h#L73). I'm not sure that there is publicly available documentation that states this.

Reply:
Please add a comment mentioning it.